pom.client.xml

```diff
@@ -543,9 +547,10 @@
               <additionalOptions>-maxLineLength 120
                 -snippetpath ${project.basedir}/sdk/appconfiguration/azure-data-appconfiguration/src/samples/java
                 -snippetpath ${project.basedir}/core/azure-core/src/samples/java
-                -snippetpath ${project.basedir}/sdk/eventhubs/azure-eventhubs/src/samples/java
-                -snippetpath ${project.basedir}/sdk/keyvault/azure-keyvault-keys/src/samples/java
-                -snippetpath ${project.basedir}/sdk/keyvault/azure-keyvault-secrets/src/samples/java
+                -snippetpath ${project.basedir}/eventhubs/client/azure-eventhubs/src/samples/java
+                -snippetpath ${project.basedir}/keyvault/client/keys/src/samples/java
+                -snippetpath ${project.basedir}/keyvault/client/secrets/src/samples/java
+                -snippetpath ${project.basedir}/storage/client/queue/src/samples/java
```

Comment:
You've reverted the lines for event hubs and keyvault to no longer be correct - the previous lines were correct. Similarly, storage queues is incorrect. Please generate the JavaDoc and validate that the code snippets are actually there as part of your pre-PR review.

Reply:
I have pasted the master pom.client.xml here and added queue stuff on top of that.